Add pop-out link to styleguide example forms

toolness opened the pull request:
Some of our example forms in the style guide--the ones that can actually be submitted, which are useful for manual testing--are also embedded as "standalone" views with their own URLs, which is particularly useful for manual testing.  This just adds a "pop-out" link that makes those views discoverable:

> ![popout](https://cloud.githubusercontent.com/assets/124687/18516256/3d1a798e-7a66-11e6-824e-9e368194a6ff.png)

The SVG happens to be taken from Font Awesome but we can really just use whatevs.

jseppi commented on styleguide/static/styleguide/vendor/font-awesome-external-link.svg:
```diff
@@ -0,0 +1 @@
+<svg width="1792" height="1792" viewBox="0 0 1792 1792" xmlns="http://www.w3.org/2000/svg"><path d="M1408 928v320q0 119-84.5 203.5t-203.5 84.5h-832q-119 0-203.5-84.5t-84.5-203.5v-832q0-119 84.5-203.5t203.5-84.5h704q14 0 23 9t9 23v64q0 14-9 23t-23 9h-704q-66 0-113 47t-47 113v832q0 66 47 113t113 47h832q66 0 113-47t47-113v-320q0-14 9-23t23-9h64q14 0 23 9t9 23zm384-864v512q0 26-19 45t-45 19-45-19l-17 ...
```
Do we need to include the Font Awesome license (or a link to it) somewhere since we are vendoring this asset within our CC0-licensed repo?

toolness replied:
Yeah, that's a good question... I wonder if there are any CC0-licensed icon libraries?  If so, I could just nab their SVG instead.

jseppi replied:
thenounproject.com is a great resource!

Here's a CC0 external link icon I just found real quick: https://thenounproject.com/search/?q=external+link&i=594418

toolness replied:
Oh nice!  Cool, I will use that.

jseppi commented:
👍 after changing out the svg

toolness commented:
Ok cool. Added the [public-domain external link SVG by Anna Debenham](https://thenounproject.com/search/?q=external+link&i=594418) and modified the `<desc>` in the SVG to credit her and link to its original location for posterity.
